cpu/atmega_common/periph/spi.c

```diff
@@ -64,7 +68,9 @@ int spi_init_master(spi_t dev, spi_conf_t conf, spi_speed_t speed)
     SPCR = ((1 << SPE) | (1 << MSTR) | conf | (speed & SPEED_MASK));
 
     /* clear interrupt flag */
-    (void)SPSR;
+    (void)SPIE;
```

Comment:
`SPIE` is the actual interrupt flag, `SPSR` was a mistake from the previous code.